### ISSUE TITLE
Discrepancy in results while fine-tuning SlowR50 on UCF101 with my BYOL pre-trained checkpoints

### ISSUE THREAD

**Issue by qinzhi-0110** [facebookresearch/SlowFast#646]
Hello,

I have been using the BYOL_SlowR50_8x8.yaml script to pre-train SlowR50 model with pBYOL on 8 machines with 8 V100 GPUs each. I modified the TRAIN_CROP_NUM_TEMPORAL parameter to 4.  Then when I fine-tuned the pre-trained model on UCF101 dataset, I achieved an accuracy of 91.72%, which is quite lower than the reported accuracy of 94.2% in the paper.

I am wondering if there are any specific techniques that I might have missed while modifying the parameter and fine-tuning the model. Could you please provide me some suggestions to improve the accuracy of the model? I would really appreciate your help in this regard.

Thank you for your time and consideration.

Best regards
